python/ray/dataframe/dataframe.py

```diff
@@ -408,6 +434,10 @@ def dtypes(self):
         Returns:
             The dtypes for this DataFrame.
         """
+        # Deal with empty column case
```

Comment:
Prefer comment to say `Deal with a DataFrame with no columns` or something along those lines. Empty columns feels a bit ambiguous (could mean all NaN in some cases).